src/components/Cart/Cart.js

```diff
@@ -18,65 +18,66 @@ import OrderSummary from '../Checkout/OrderSummary';
-                <OrderSummary checkout={true} />
-                <Text>
-                  Or,{' '}
-                  <Link
-                    as={RouterLink}
-                    color={colors.primary}
-                    to='/product/product-listing'
-                  >
-                    continue shopping
-                  </Link>
-                </Text>
-              </VStack>
-            </GridItem>
-          </SimpleGrid>
-        </Flex>
-      </Elements>
-    </div>
-  );
+    return (
+        <div data-testid='cart '>
+            <Elements stripe={stripePromise}>
+                <Flex h='100vh' py={20}>
+                    <SimpleGrid columns={3} rowGap={6} columnGap={3} w='full'>
+                        <GridItem colSpan={2}>
+                            <Box
+                                w='full'
+                                h='full'
+                                p={5}
+                                spacing={10}
+                                borderWidth='2px'
+                                borderRadius='25px'
+                            >
+                                <Heading
+                                    p={4}
+                                    mb={6}
+                                >{`Shopping Cart (${calculateTotalItemsInCart()} items)`}</Heading>
+                                {cart.map((product) => (
+                                    <CartItem
+                                        key={product.id}
+                                        id={product.id}
+                                        title={product.title}
+                                        description={product.description}
+                                        unitPrice={product.unitPrice}
+                                        itemQuantity={product.quantity}
+                                        imageURL={product.imageURL}
+                                        altImage={product.altImage}
+                                    />
+                                ))}
+                            </Box>
+                        </GridItem>
+                        <GridItem colSpan={1}>
+                            <VStack>
+                                <OrderSummary checkout={true} />
+                                <Text>
+                                    Or,{' '}
+                                    <Link
+                                        as={RouterLink}
+                                        color={colors.primary}
+                                        to='/products'
+                                    >
+                                        continue shopping
+                                    </Link>
+                                </Text>
+                            </VStack>
+                        </GridItem>
+                    </SimpleGrid>
+                </Flex>
+            </Elements>
+        </div>
+    );
 };
 
 export default Cart;
```

Comment:
This is the entire cart page right? Maybe it would be better to move to `src/pages`

Reply:
yes its the entire cart page. Ok i will move it.